Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -293,6 +316,14 @@ extension CharcoalViewController: UINavigationControllerDelegate {
     }
 }
 
+// MARK: - MapTabBarControllerDelegate
+
+extension CharcoalViewController: MapTabBarControllerDelegate {
+    func mapTabBarController(_ mapTabBarController: MapTabBarController, didSelect selection: CharcoalViewController.PolygonSelection) {
+        selectionDelegate?.charcoalViewController(self, didSelect: selection)
```

Comment:
I'm not quite sure why we need a specialized `didSelect` method for polygon? Because there is no similar for the radius filter being selected? Or maybe I just didn't see that :) 

Perhaps it will be clear once I see how it is integrated.

Reply:
It's basically just for tracking. I added this enum, so that I don't have to add a new function for everything I want to track in the map search. Maybe we should just have a more generic `TrackingEvent` instead? Or call it something else.

Reply:
Aha, ok, then I understand. 

Probably fine to do this, we should maybe change to something generic if we need it for more things.